test/build.info

```diff
@@ -400,7 +400,8 @@ INCLUDE_MAIN___test_libtestutil_OLB = /INCLUDE=MAIN
   # names with the DLL import libraries.
   IF[{- $disabled{shared} || $target{build_scheme}->[1] ne 'windows' -}]
```

Comment:
This line says don't compile this test if we're on shared library Windows build - which is why you're getting the Appveyor failure when it tries to run it.